bundles/org.openhab.binding.logreader/README.md

```diff
@@ -98,6 +98,17 @@ rule "LogReader"
     end
 ```
 
+Use the rules with your Telegram Bot (need openHAB Telegram Action installed and configured)
```

Comment:
Please add one empty line between the sentence and the code block.

Reply:
done :)